python-threatexchange/threatexchange/tests/hashing/test_pdq_hasher.py

```diff
@@ -59,7 +59,15 @@ def test_pdq_from_file(self):
             assert pdq_hash == RANDOM_IMAGE_PDQ
 
     def test_pdq_from_bytes(self):
-        """Runs the pdq hashser directly on bytes"""
+        """Runs the pdq hasher directly on bytes"""
         bytes_ = base64.b64decode(RANDOM_IMAGE_BASE64)
         pdq_hash = pdq_hasher.pdq_from_bytes(bytes_)[0]
         assert pdq_hash == RANDOM_IMAGE_PDQ
+
+    def test_pdq_from_file_la_png(self):
```

Comment:
(ignorable): For goofballs like me that have never seen la before, can you add a dobloc to reference "LA images (luminance with alpha)"